app/services/user_event_creator.rb

```diff
@@ -8,13 +8,19 @@ def initialize(current_user:, request: nil)
     @current_user = current_user
   end
 
-  def create_user_event(event_type, user = current_user)
+  def create_user_event(event_type, user = current_user, disavowal_token = nil)
     return unless user&.id
     existing_device = Device.find_by(user_id: user.id, cookie_uuid: cookies[:device])
     if existing_device.present?
-      create_event_for_existing_device(event_type: event_type, user: user, device: existing_device)
+      create_event_for_existing_device(
+        event_type: event_type, user: user, device: existing_device,
+        disavowal_token: disavowal_token
```

Comment:
i think rubocop will request a trailing comma?
```suggestion
        disavowal_token: disavowal_token,
```